.github/workflows/test.yml

```diff
@@ -26,6 +26,8 @@ jobs:
     name: ${{ matrix.os }} Py-${{ matrix.python }} IPFS-${{ matrix.ipfs }}
     steps:
       - uses: actions/checkout@v2
+      - name: Echo branch being used
+        run: git branch
```

Comment:
Do we need this step?

Reply:
That is the aforementioned sanity check. Not necessary but useful in the currently running GitHub action to validate the "default behavior". If the default behavior is, in fact, to use the PR commit instead of the main branch, I will remove it.